packages/cozy-sharing/package.json

```diff
@@ -41,7 +41,7 @@
     "babel-jest": "26.6.3",
     "babel-plugin-css-modules-transform": "^1.6.2",
     "babel-plugin-inline-react-svg": "^1.1.0",
-    "cozy-client": "16.0.0",
+    "cozy-client": "16.8.0",
```

Comment:
If you take a look at the definition, you'll see that this change is not a "chore" commit https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type

"feat: Upgrade cozy-client to have access to partialIndex" will be more meaningful

Reply:
Fair enough :+1: